### ISSUE TITLE
J2V8 native library not loaded

### ISSUE THREAD

**Issue by djx314** [eclipsesource/J2V8#260]
Open this issue though there are some issue have the same problem with it because I use j2v8 in scala and server site.

I use sbt and [playframework](https://github.com/playframework/playframework) to build my website. And I use j2v8 and dust.js to render my web page. When I use
```shell
sbt>
run
ctrl + D to stop my application
run
```

and then the website will throw an exception:
```scala
Caused by: java.lang.IllegalStateException: J2V8 native library not loaded
        at com.eclipsesource.v8.V8.checkNativeLibraryLoaded(V8.java:195)
        at com.eclipsesource.v8.V8.createV8Runtime(V8.java:149)
        at com.eclipsesource.v8.V8.createV8Runtime(V8.java:125)
        at com.eclipsesource.v8.NodeJS.createNodeJS(NodeJS.java:58)
        at com.eclipsesource.v8.NodeJS.createNodeJS(NodeJS.java:45)
        at assist.utils.DustEngine$$anonfun$1.apply(DustEngine.scala:48)
        at assist.utils.DustEngine$$anonfun$1.apply(DustEngine.scala:47)
        at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(
Future.scala:24)
        at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.sca
la:24)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.
java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor
.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.UnsatisfiedLinkError: Could not load J2V8 library. Reasons:

        no j2v8_win32_x86_64 in java.library.path

        at com.eclipsesource.v8.LibraryLoader.loadLibrary(LibraryLoader.java:75)

        at com.eclipsesource.v8.V8.load(V8.java:71)
        at com.eclipsesource.v8.V8.createV8Runtime(V8.java:145)
        ... 10 common frames omitted
```

And I start another project use j2v8 too. It will throw this exception too.

It seems that j2v8 can only run one application in one computer(both in windows and linux).

Any fix? I think the file ~/libj2v8_win32_x86_64.dll is the problem.

Will 4.7 slove this problem?

**Comment by irbull** [comment]
I think this is a duplicate of https://github.com/eclipsesource/J2V8/issues/39.

**Comment by djx314** [comment]
Yes, I think so. Thanks! I will look out issue 39 for any update.

**Comment by irbull** [comment]
Since we have #39, I'm going to close this. Right now several of us are working on the build system for the native parts. Once we have the build system updated, we should be able to version the native libs, and thus fix #39.
